config/ci.js

```diff
@@ -3,10 +3,7 @@ const AWS = require('aws-sdk')
 module.exports = {
   aws: {
     s3: {
-      endpoint: new AWS.Endpoint(
-        // randomise port to avoid conflicts in parallel test runs
-        `http://localhost:${Math.floor(Math.random() * 50000) + 15000}`,
-      ),
+      endpoint: new AWS.Endpoint('http://localhost:4569'),
```

Comment:
Is localhost:4569 mapped to the fakes3 pod in Kubernets?

Reply:
Yes, `localhost` cover the whole pod which hosts both `elife-xpub` and `fakes3`:
https://github.com/elifesciences/elife-xpub-formula/blob/master/helm/elife-xpub/templates/deployment.yaml#L73

Reply:
For a testing environment it wasn't worth it to have a separate Deployment and Service